src/web-auth/transaction-manager.js

```diff
@@ -11,20 +11,21 @@ function TransactionManager(options) {
 
 TransactionManager.prototype.process = function(options) {
   var transaction;
+  var responseType = options.responseType || '';
 
-  if (options.responseType.indexOf('code') !== -1) {
+  if (responseType.indexOf('code') !== -1) {
     return options;
   }
 
-  if (options.responseType.indexOf('id_token') !== -1 && !!options.nonce) {
+  if (responseType.indexOf('id_token') !== -1 && !!options.nonce) {
```

Comment:
what will happen if `responseType=id_token` and no nonce set?

Reply:
return raw options